package/test-package.sh

```diff
@@ -43,3 +43,9 @@ kevm prove tests/specs/benchmarks/structarg00-spec.k         \
     --save-directory proofs                                  \
     --verbose                                                \
     --use-booster
+
+kevm prove tests/specs/benchmarks/structarg01-spec.k         \
+    --definition tests/specs/benchmarks/verification/haskell \
+    --save-directory proofs                                  \
+    --verbose                                                \
+    --use-booster
```

Comment:
What's the reason for adding this test as a smoke test, rather than as a functional test to the `pytest` harness?

Reply:
No specific reason, I saw the previous test for `structarg00` being added like this.

Reply:
In that case, please open an issue for porting these tests to Python.

Reply:
Opened https://github.com/runtimeverification/evm-semantics/issues/2275.